GOVERNANCE.md

```diff
@@ -0,0 +1,51 @@
+# Pinniped Governance
+
+This document defines the project governance for Pinniped.
+
+# Overview
+
+**Pinniped** is committed to building an open, inclusive, productive and self-governing open source community focused on building authentication services for Kubernetes clusters. The
+community is governed by this document which defines how all members should work together to achieve this goal.
+
+# Code of Conduct
+
+The Pinniped community abides by this [code of conduct]([CODE_OF_CONDUCT.md](https://github.com/vmware-tanzu/pinniped/blob/main/CODE_OF_CONDUCT.md)).
+
+# Community Roles
+
+* **Users:** Members that engage with the Pinniped community via any medium (Slack, GitHub, mailing lists, etc.).
+* **Contributors:** Do regular contributions to the Pinniped project (documentation, code reviews, responding to issues, participating in proposal discussions, contributing code, etc.).
+* **Maintainers:** Responsible for the overall health and direction of the project. They are the final reviewers of PRs and responsible for Pinniped releases.
+
+# Maintainers
+New maintainers must be nominated by an existing maintainer and must be elected by a supermajority of existing maintainers. Likewise, maintainers can be removed by a supermajority of the existing maintainers or can resign by notifying one of the maintainers.
+
+**Note:** If a maintainer leaves their employer they are still considered a maintainer of Pinniped, unless they voluntarily resign. Employment is not taken into consideration when determining maintainer eligibility unless the company itself violates our [Code of Conduct](https://github.com/microwavables/pinniped/blob/main/CODE_OF_CONDUCT.md).
```

Comment:
Looks like the link is pointing at your fork rather than the base repo
```suggestion
**Note:** If a maintainer leaves their employer they are still considered a maintainer of Pinniped, unless they voluntarily resign. Employment is not taken into consideration when determining maintainer eligibility unless the company itself violates our [Code of Conduct](https://github.com/vmware-tanzu/pinniped/blob/main/CODE_OF_CONDUCT.md).
```

Reply:
Good catch @margocrawf !! Committed :)